getdist/plots.py

```diff
@@ -1085,6 +1085,12 @@ def clean_args(_args):  # prevent unused argument warnings
             _args.pop('lw', None)
             return _args
 
+        args = self._get_line_styles(plotno, **kwargs)
+        linestyles = [args['ls']]
```

Comment:
I think _get_line_styles always returns args['lw'] so with this change self.settings.linewidth_contour is never used?